docs/platform/concepts/database-forking.rst

```diff
@@ -15,13 +15,17 @@ When you fork a service, the following items are copied into the new service:
 
 You can fork the following Aiven services:
 
-- PostgreSQL
+- PostgreSQL®
 - MySQL
-- Redis
-- Cassandra (Limitation: you cannot fork to a lower amount of nodes)
+- Redis™*
```

Comment:
Why is there an `*`?

Reply:
Clause 6.a.i of "GENERAL USE INFORMATION" at https://redis.io/topics/trademark tells us to use the™ and the *